staging_service/app.py

```diff
@@ -122,17 +130,17 @@ async def bulk_specification(request: web.Request) -> web.json_response:
     Data Transfer Service manifest.
     """
     username = await authorize_request(request)
-    files = parse_qs(request.query_string).get("files", [])
-    files = files[0].split(",") if files else []
-    files = [f.strip() for f in files if f.strip()]
-    paths = {}
-    for f in files:
-        p = Path.validate_path(username, f)
-        paths[PathPy(p.full_path)] = PathPy(p.user_path)
+    params = parse_qs(request.query_string)
+    paths = _extract_file_paths(params, "files", username)
+    as_dts = params.get("dts") == "1"
+
     # list(dict) returns a list of the dict keys in insertion order (py3.7+)
+    file_type_resolver = _file_type_resolver
+    if as_dts:
+        file_type_resolver = lambda: FileTypeResolution(parser=parse_dts_manifest)  # noqa: E731
```

Comment:
This is the core of the change, really.
If as_dts is set (and the files exist), just always use `parse_dts_manifest`. A little shaky, maybe, since a user could flag any non-JSON file this way, but then they're silly and should see an error anyway.

Reply:
noqa is in there because Ruff doesn't like inline lambdas and is wrong.